docs/samples/Microsoft.ML.AutoML.Samples/MulticlassClassificationExperiment.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Auto;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.AutoML.Samples
+{
+    public static class MulticlassClassificationExperiment
+    {
+        private static string BaseDatasetsLocation = "Data";
+        private static string TrainDataPath = Path.Combine(BaseDatasetsLocation, "optdigits-train.csv");
+        private static string TestDataPath = Path.Combine(BaseDatasetsLocation, "optdigits-test.csv");
+        private static string ModelPath = Path.Combine(BaseDatasetsLocation, "OptDigits.zip");
+        private static string LabelColumnName = "Number";
+        private static uint ExperimentTime = 60;
+
+        public static void Run()
+        {
+            MLContext mlContext = new MLContext();
+
+            // STEP 1: Load data
+            IDataView trainDataView = mlContext.Data.LoadFromTextFile<PixelData>(TrainDataPath, hasHeader: true, separatorChar: ',');
+            IDataView testDataView = mlContext.Data.LoadFromTextFile<PixelData>(TestDataPath, hasHeader: true, separatorChar: ',');
+
+            // STEP 2: Run AutoML experiment
+            Console.WriteLine($"Running AutoML multiclass classification experiment for {ExperimentTime} seconds...");
+            ExperimentResult<MulticlassClassificationMetrics> experimentResult = mlContext.Auto()
+                .CreateMulticlassClassificationExperiment(ExperimentTime)
+                .Execute(trainDataView, LabelColumnName);
+
+            // STEP 3: Print metric from the best model
+            RunDetail<MulticlassClassificationMetrics> best = experimentResult.BestRun;
+            Console.WriteLine($"Total models produced: {experimentResult.RunDetails.Count()}");
+            Console.WriteLine($"Best model's trainer: {best.TrainerName}");
+            Console.WriteLine($"AccuracyMacro of best model from validation data: {best.ValidationMetrics.MacroAccuracy}");
```

Comment:
Why MacroAccuracy? Generally, I'd just print all metrics.

If we're going to display MacroAccracy, I'd recommend training towards it (or a proxy metric). Macro & Micro accuracy are sometimes at odds, for instance adding a weight column to balance the effect of each class tends to increase macro accuracy at the cost of micro.

Reply:
You're right -- I copied this part over from the old repo. Thanks. Let me print MicroAccuracy, since we're sweeping to optimize it.
Re: printing all metrics -- in our draft into the samples repo, where we assume the user will execute the sample code, we have a helper function that prints all the metrics like you suggest. The example code here will appear in the API docs for the user to view, but may not be targeted for stand-alone runs. Given this, do you think it would be helpful to print all metrics? If so, I can go ahead and do this for the samples here

Reply:
How obvious is helper function? For clarity, we could have the 4 print statements directly.

I like seeing training time for my runs; do we pass that back to the user?

Reply:
rev-ed multiclass sample https://github.com/dotnet/machinelearning/blob/4b5bfd5eb69e9026b07bb98f18e3ed2268094e45/docs/samples/Microsoft.ML.AutoML.Samples/MulticlassClassificationExperiment.cs#L43
I can update metrics printing for remaining samples if that looks alright
I agree that printing iteration time for each model would be nice to have. Let's brainstorm together with Vinod all the other things the samples here should include, and what they should look like

Reply:
for the sake of time, going to iterate on metrics printing for Regression & Binary in the same way, and then check-in. if there's a different way that would be preferable, will submit a PR to update